system/config/countries.php

```diff
@@ -56,7 +57,7 @@
 	'cf' => 'Central African Republic',
 	'cg' => 'Congo, Republic of',
 	'ch' => 'Switzerland',
-	'ci' => 'Cote d\'Ivoire',
+	'ci' => 'Côte d\'Ivoire',
```

Comment:
AFAIR we only had lower ASCII chars in this file so far, the UTF-8 chars are noted in the language files only.
By changing this now, we are IMO introducing a breaking change here.